tests/Controllers/OAuth/TokensControllerTest.php

```diff
@@ -36,4 +49,76 @@ public function testDestroyCurrentClientGrant()
 
         $this->assertTrue($token->fresh()->revoked);
     }
+
+    public function testIssueTokenWithPassword(): void
+    {
+        \Mail::fake();
+
+        $user = User::factory()->create();
+        $client = (new ClientFactory())->create([
+            'password_client' => true,
+        ]);
+
+        $this->expectCountChange(fn () => $user->tokens()->count(), 1);
+
+        $tokenResp = $this->json('POST', route('oauth.passport.token'), [
+            'grant_type' => 'password',
+            'client_id' => $client->getKey(),
+            'client_secret' => $client->secret,
+            'scope' => '*',
+            'username' => $user->username,
+            'password' => UserFactory::DEFAULT_PASSWORD,
+        ])->assertSuccessful();
+        $tokenJson = json_decode($tokenResp->getContent(), true);
+
+        $meResp = $this->json('GET', route('api.me'), [], [
+            'Authorization' => "Bearer {$tokenJson['access_token']}",
+        ])->assertSuccessful();
+        $meJson = json_decode($meResp->getContent(), true);
```

Comment:
There's this `TestResponse::decodeResponseJson()` and `assertPath`/`assertJsonPath` thing